modules/lss-missionHelper/missionHelper.user.js

```diff
@@ -633,7 +633,10 @@
 				364 : '1x Löschfahrzeug<br><br><span class="badge">Credits: ~ 500 </span>',
 				365 : '1x Löschfahrzeug<br><br><span class="badge">Credits: ~ 500 </span>',
 				366 : '2x Funkstreifenwagen<br><br><span class="badge">Credits: ~ 550 </span>',
-				367 : '6x Funkstreifenwagen<br>4x SEK-Fahrzeuge<br>1x FüKw<br><br><span class="badge">Credits: ~ 2800 </span>'
+				367 : '6x Funkstreifenwagen<br>4x SEK-Fahrzeuge<br>1x FüKw<br><br><span class="badge">Credits: ~ 2800 </span>',
+				368 : '15x Löschfahrzeuge<br>3x Drehleitern<br>2x Schlauchwagen<br>2x GW-Atemschutz<br>4y Rüstwagen<br>3x Einsatzleitwagen 1<br>1x Einsatzleitwagen 2' +
```

Comment:
4y Rüstwagen
ich glaub, das "y" sollte ein "x" sein ;)

Reply:
ups das liegt daran das x und y nebeneinander liegen :D is fixxed